src/pytest_cov/engine.py

```diff
@@ -67,13 +67,16 @@ def _data_suffix(name):
 class CovController:
     """Base class for different plugin implementations."""
 
-    def __init__(self, cov_source, cov_report, cov_config, cov_append, cov_branch, config=None, nodeid=None):
+    cov: coverage.Coverage | None
```

Comment:
Perhaps move this into the initializer? Any reason to have it right here?

Reply:
It's just a type annotation - ":" doesn't assign anything to the "cov" class attribute. I added it mostly to make some things easier in my editor.